schemas/error.yml

```diff
@@ -29,6 +29,12 @@
       description: >
         Error code describing the error.
 
+    - name: type
+      level: core
```

Comment:
I kind of agree that this should be core but in general I would try to have as little field as possible in core as we will not get them "out there" anymore :-) Could we make it extended for now?